contracts/core/BaseAccount.sol

```diff
@@ -31,6 +41,64 @@ abstract contract BaseAccount is IAccount {
      */
     function entryPoint() public view virtual returns (IEntryPoint);
 
+    /**
+     * execute a single call from the account.
+     */
+    function execute(address target, uint256 value, bytes calldata data) virtual external {
```

Comment:
I have to note that this `execute` function is a strict subset of what `executeBatch` can do. It is trivially easy for any wallet to always make a call to `executeBatch` with a single call.
There is also no large legacy infrastructure around `execute(address,uint256,bytes)` functions in smart contract wallets.
I know we always had it in the Sample Account, but moving both of them to the **core/**`BaseAccount`  is a little bit too much.
I vote we drop the single `execute` and only leave `executeBatch`.

Reply:
execute is slightly more efficient, and I can see wallets do use it, when a batch is not required.